### DISCUSSION TITLE
Microsoft Dev Account

### DISCUSSION THREAD

**Discussion by iamLazyCode** [iterate-ch/cyberduck#16540] [Q&A]
Microsoft Dev Account 5tb shows 1tb instead , does anyone know why and if i can fix it somehow ?

**Comment by dkocher** [comment]
It depends on [quota support](https://docs.mountainduck.io/mountainduck/connect/#quota-support) if it is possible to show actual values that are provided by the storage provider.

**Reply by iamLazyCode** [reply]
> It depends on [quota support](https://docs.mountainduck.io/mountainduck/connect/#quota-support) if it is possible to show actual values that are provided by the storage provider.

Thankyou sir so is it configurable or will always be limited

**Reply by tatze96** [reply]
The displayed disk space is limited to the remaining space within the local synchronization cache location for protocols and servers without quota support while using the _Smart Synchronization_ mode. To work around this, you can either change the [cache location](https://docs.mountainduck.io/mountainduck/preferences/#cache-location) to a destination with more free disk space or use the _Online_ mode instead of the _Smart Synchronization_ mode to get exabyte values displayed.
